patches/minecraft/net/minecraft/client/multiplayer/ClientChunkProvider.java.patch

```diff
@@ -1,18 +1,10 @@
 --- a/net/minecraft/client/multiplayer/ClientChunkProvider.java
 +++ b/net/minecraft/client/multiplayer/ClientChunkProvider.java
-@@ -61,6 +61,7 @@
-          int i = this.field_217256_d.func_217191_a(p_73234_1_, p_73234_2_);
-          Chunk chunk = this.field_217256_d.func_217192_a(i);
-          if (func_217249_a(chunk, p_73234_1_, p_73234_2_)) {
-+            net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.world.ChunkEvent.Unload(chunk));
-             this.field_217256_d.func_217190_a(i, chunk, (Chunk)null);
+@@ -100,6 +100,7 @@
+             chunk = new Chunk(p_217250_1_, new ChunkPos(p_217250_2_, p_217250_3_), new Biome[256]);
+             chunk.func_217326_a(p_217250_4_, p_217250_5_, p_217250_6_, p_217250_7_);
+             this.field_217256_d.func_217181_a(i, chunk);
++            net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.world.ChunkEvent.Load(chunk));
+          } else {
+             chunk.func_217326_a(p_217250_4_, p_217250_5_, p_217250_6_, p_217250_7_);
           }
- 
-@@ -113,6 +114,7 @@
-             worldlightmanager.func_215566_a(SectionPos.func_218154_a(p_217250_2_, j, p_217250_3_), ChunkSection.func_222628_a(chunksection));
-          }
- 
-+         net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.world.ChunkEvent.Load(chunk));
-          return chunk;
-       }
```

Comment:
This move is also useless, as it is the same code flow, but you moved it above the chunk being finalized/lit for some unknown reason.